### ISSUE TITLE
Steps to Build Example From Clean Clone?

### ISSUE THREAD

**Issue by perlmunger** [FolioReader/FolioReaderKit#118]
The documentation says to use 'pod install', however doing this from the root directory produces this error:

> [!] No `Podfile' found in the project directory.

Running it in the Example directory produces this:

> [!] Unable to find a specification for `Quick`

What is the magical incantation to get the example app to build? Thanks.

-Matt

**Comment by kjantzer** [comment]
You're correct to run it from the example directory, but I'm not sure how to fix the specification error.

Maybe related to: http://stackoverflow.com/questions/25913733/unable-to-find-a-specification-in-cocoapods

**Comment by perlmunger** [comment]
@kjantzer So are you saying there is something wrong with my setup? I'm confused why building from a clean clone doesn't just work.

**Comment by kjantzer** [comment]
@perlmunger – Maybe try this? http://stackoverflow.com/a/28919637/484780
